### PR TITLE
Convert Code of Conduct to Markdown

### PR DESCRIPTION
**Pull request by ajyong**
Waiting on #44 and #45 to complete first.  I switched the template it uses over to `default`.  We lose the snazziness but maintain Jekyll project correctness.

image

### CODE REVIEW

File: policies/rules.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ Competition in this Hackathon officially begins on March 16th at 00:00 UTC and c
...
+All participants, organizers, sponsors and contributors agree to follow the CODEVID-19 [Code of Conduct](code-of-conduct.html) and understand that failure to do so could result in consequences up to and including elimination and exclusion from competition, Slack, and other resources provided along with forfeiture of any prize winnings or consideration provided by CODEVID-19, the Dev Edmonton Society, and associated parties. Participants understand that enforcing and interpreting the [Code of Conduct](code-of-conduct.html) is done at the sole discretion of the CODEVID-19 organizing volunteers. Questions regarding the enforcement or interpretation of the [Code of Conduct](code-of-conduct.html) should be directed to [conduct@codevid19.com](mailto:conduct@codevid19.com).
```
Comment (ajyong) [inline policies/rules.md:141]: The only changes here were to add hyperlinks to the 2 other spots in this paragraph that refer to the CoC.

**Comment by ajyong** [comment]
I've removed Draft status (I have never used this before) and it's ready for review

File: index.html [hunk 2]
```diff
@@ -102,7 +102,9 @@ <h3>Developers & Designers</h3>
         </p>
         <p>
           Remember to read the rules and
-          <a href="code-of-conduct.html"> Code of Conduct</a>
+          <a href="{{ '/policies/code-of-conduct.html' | relative_url }}">
```
Comment (MarkBennett) [inline index.html:105]: :+1:

File: index.html [hunk 3]
```diff
@@ -227,7 +229,9 @@ <h2>Links and Resources</h2>
           <h3>Code of Conduct</h3>
           <p>
             All participants, humans and bots must abide by this
-            <a href="code-of-conduct.html">Code of Conduct.</a>
+            <a href="{{ '/policies/code-of-conduct.html' | relative_url }}"
```
Comment (MarkBennett) [inline index.html:232]: :+1:

**Review by MarkBennett — APPROVED** [review]
Looks good @ajyong!

**Pull request merged**